src/applications/hca/config/form.js

```diff
@@ -250,77 +125,15 @@ const formConfig = {
           path: 'veteran-information/personal-information',
           title: 'Veteran information',
           initialData: {},
```

Comment:
are we using the `initialData: {}` prop or can this be removed?

Reply:
Looks like if I remove those it triggers contract errors for 508 in CY Axe Check